.github/workflows/macos-filebeat.yml

```diff
@@ -0,0 +1,27 @@
+name: filebeat
+
+on:
+  push:
+    branches:
+      - master
+      - 7.1*
+      - 8.*
+
+env:
+  BEAT_MODULE: 'filebeat'
+
+jobs:
+  macos:
+    runs-on: macos-latest
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.17.5'
```

Comment:
Not sure if it's possible to read .go-versión file in here, to avoid multiple occurrences of the Go version

Reply:
That's a good point, when I looked for something like that I was not able to find any references, let me search again

Reply:
https://github.com/actions/setup-go/issues/23 seems the one that requested something similar, in fact a PR was created sometime ago but it has not been merged yet.

I'll try with https://github.com/actions/setup-go/issues/23#issuecomment-732276072

Reply:
This is now solved in https://github.com/elastic/beats/pull/29032/commits/af87f1dd1b37788b805038aebb0716675bc440f3 and https://github.com/elastic/beats/pull/29032/commits/7ac9f4f0232a2d740344d01ec3d7ea227f7755f6